gapic-generator/templates/default/service/client/method/docs/_request_normal.erb

```diff
@@ -3,5 +3,12 @@
 <%- if method.doc_description -%>
 <%= indent method.doc_description, "#   " %>
 <%- end -%>
-# @param options [Google::Gax::ApiCall::Options, Hash]
+#
+#   When using a hash, the following fields are supported:
+#
+<%- method.arguments.each do |arg| -%>
+#   * `<%= arg.name %>` (`<%= arg.doc_types %>`)<%- if arg.doc_description -%>:
```

Comment:
What do you think about using square brackets for the types list, like YARD does?

Reply:
This is markdown content, and not using YARD markup. Users won't ever see the square brackets used by YARD. I thought it would make more sense to see the type in parens on the generated documentation.